src/silx/gui/plot/items/image_aggregated.py

```diff
@@ -180,7 +180,7 @@ def _getLevelOfDetails(self) -> Tuple[int, int]:
         return lodx, lody
 
     @docstring(ImageDataBase)
-    def setData(self, data, copy=True):
+    def setData(self, data, copy=True, **kwargs):
```

Comment:
Why is `**kwargs` needed?

Reply:
Without the kwargs, we got this error for ImageAggregate.setdata
```
ERROR:silx.app.view.main:An error occured in silx view:
ERROR:silx.app.view.main:<class 'TypeError'> ImageDataAggregated.setData() got an unexpected keyword argument 'alternative'   File "/home/edgar1993a/work/silx/src/silx/gui/data/DataViewer.py", line 299, in __setDataInView
    self.__currentView.setData(self.__displayedData)
  File "/home/edgar1993a/work/silx/src/silx/gui/data/DataViews.py", line 673, in setData
    self.__currentView.setData(data)
  File "/home/edgar1993a/work/silx/src/silx/gui/data/DataViews.py", line 1123, in setData
    plot.addImage(
  File "/home/edgar1993a/work/silx/src/silx/gui/plot/PlotWidget.py", line 1496, in addImage
    image.setData(data, alternative=pixmap, copy=copy)
```

Reply:
The `**kwargs` should not be needed since there is no need to call `addImage` in the DataView.